docs/navigation-configuration.md

```diff
@@ -66,6 +74,23 @@ You can configure the way Luigi tackles routing in your application in the **Rou
 - **nodeAccessibilityResolver** allows you to define a permission checker function that gets executed on every node. If it returns `false`, Luigi removes the node and its children from the navigation structure.
   **nodeAccessibilityResolver** receives all values defined in the node configuration. See [angular sampleconfig.js](../core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js) for the **constraints** example.
 
+## Context switcher
+
+the context switcher is a drop-down list available in the top navigation bar. It allows you to switch between a curated list of navigation elements such as Environments.
+
+- **defaultLabel** specifies the default label that is shown if no context is selected.
+- **parentNodePath** specifies the base path, that is prepended to **options[].pathValue**. It must be an absolute path.
+- **lazyloadOptions** defines when to fetch **options**. When set to `true`, loads **options** when you click the context switcher. It doesn't involve any caching. When set to `false`, loads **options** once when the page loads . The default value is `true`. 
+- **options** defines the list of context element. Context element properties are:
+  - **label** defines the context element label. If not defined, the **pathValue** is passed to **fallbackLabelResolver** to set its value. The default value is **pathValue**, if **fallbackLabelResolver** is not defined.
+  - **pathValue** defines the context element path that is appended to **parentNodePath** and reflects a **pathSegment**.
+- **actions** defines a list of additional elements that are shown on above or below the context switcher **options**. Each action contains the following parameters:
+  - **label** defines the action element label
+  - **position** is optional. Defines the action element position. Can be `top` or `bottom`. The default value is `top.`
```

Comment:
`Defines the action element position. Can be `top` or `bottom`. The default value is `top.` This parameter is optional.`

I moved the info about it being optional to the end, to be more consistent in the way we are describing the parameters.